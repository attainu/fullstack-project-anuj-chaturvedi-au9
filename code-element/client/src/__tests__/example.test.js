
import React from 'react'
import { shallow , mount} from 'enzyme'
import Example from '../components/layout/example'

     
    it('it renders Example',()=>{
        const wrapper = shallow(<Example/>);
        expect(wrapper).toMatchSnapshot();
    });

    it('it renders Example element',()=>{
        const wrapper = mount(<Example/>);
        
        expect(wrapper.find('.exampleImage')).toMatchSnapshot();
    });
    
    

