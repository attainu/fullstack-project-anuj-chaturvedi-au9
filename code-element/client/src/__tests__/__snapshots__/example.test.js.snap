// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders Example 1`] = `ShallowWrapper {}`;

exports[`it renders Example element 1`] = `ReactWrapper {}`;
